src/screens/DomainScreen/elements/Header/index.js

```diff
@@ -127,25 +127,6 @@ const styles = StyleSheet.create({
   container: {
     flexDirection: 'row'
   },
-  image: {height: '100%', width: '100%', borderRadius: 45},
-  containerImage: {flex: 1.3},
-  boxImage: {
-    borderRadius: 45,
-    borderWidth: 0.2,
-    borderColor: COLORS.black50,
-    width: 60,
-    height: 60,
-    justifyContent: 'center',
-    alignItems: 'center'
-  },
-  buttonPrimary: {
-    height: normalize(36),
-    backgroundColor: COLORS.anon_primary,
-    // flex: 1,
-    justifyContent: 'center',
-    alignItems: 'center',
-    borderRadius: 8
-  },
   wrapperHeader: {
     flex: 1,
     justifyContent: 'flex-start',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here seem to be removing some styles and adding others. It's hard to evaluate these changes without seeing the full context of how these styles are used. If these changes are intentional and tested, they should be fine. However, please ensure that these changes do not break the layout or appearance of your components.
```diff
-  image: {height: '100%', width: '100%', borderRadius: 45},
-  containerImage: {flex: 1.3},
-  boxImage: {
-    borderRadius: 45,
-    borderWidth: 0.2,
-    borderColor: COLORS.black50,
-    width: 60,
-    height: 60,
-    justifyContent: 'center',
-    alignItems: 'center'
-  },
-  buttonPrimary: {
-    height: normalize(36),
-    backgroundColor: COLORS.anon_primary,
-    // flex: 1,
-    justifyContent: 'center',
-    alignItems: 'center',
-    borderRadius: 8
-  },
+  container: {
+    flexDirection: 'row'
+  },
+  wrapperHeader: {
+    flex: 1,
+    justifyContent: 'flex-start',
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->